app/views/routines/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="">
+  <div class="">
+    <strong>Usuario:</strong> <%= @routine.user.name %>
+  </div>
+  <div class="">
+    <strong>Ejercicio:</strong> <%= @routine.exercise %>
+  </div>
+</div>
```

Comment:
Why do the divs have empty class attributes 
Just delete them 
Also mind the usage of divs 
It makes your seo worse and isn't accessible